src/pysweepme/Ports.py

```diff
@@ -454,13 +486,26 @@ def find_resources_internal(self):
         return resources
 
 
+class PureSocket(PortType):
```

Comment:
To make it more similar with other ports like GPIB, TPCIP, USBTMC, please rename to SOCKET instead of PureSocket